homeassistant/components/nws/sensor.py

```diff
@@ -33,13 +40,121 @@
     DOMAIN,
     NWS_DATA,
     OBSERVATION_VALID_TIME,
-    SENSOR_TYPES,
-    NWSSensorEntityDescription,
 )
 
 PARALLEL_UPDATES = 0
 
 
+@dataclass
+class NWSSensorEntityDescription(SensorEntityDescription):
+    """Class describing NWSSensor entities."""
+
+    unit_convert: str | None = None
+
+
+SENSOR_TYPES: tuple[NWSSensorEntityDescription, ...] = (
+    NWSSensorEntityDescription(
+        key="dewpoint",
+        name="Dew Point",
+        icon=None,
```

Comment:
Can you please remove properties when they match the default?
```suggestion
```

Please remove for `icon`, for `device_class` and for `state_class`

Reply:
Done.